fk/main.go

```diff
@@ -144,57 +144,63 @@ Options:
 	return code
 }
 
-func ensureCrontabStateMatchesConfig() {
+func ensureSchedulerStateMatchesConfig() {
 	shouldEnable, err := shouldEnableScheduler()
 	if err != nil {
 		log.Panic(err)
 	}
 
 	if shouldEnable {
-		crontabWasAdded, err := scheduler.Enable(nil)
+		schedulerEnabled, err := scheduler.Enable()
```

Comment:
I think dropping the `Was` word lost some nuance. Further on in the code, `schedulerEnabled` could easily be taken as "is the scheduler enabled?". But it doesn't mean that, it means "Was it disabled before, and now we enabled it?"

Reply:
Agreed, I'll reintroduce that.